airflow/providers/databricks/operators/databricks.py

```diff
@@ -29,6 +29,7 @@
 from airflow.configuration import conf
 from airflow.exceptions import AirflowException, AirflowProviderDeprecationWarning
 from airflow.models import BaseOperator, BaseOperatorLink, XCom
+from airflow.operators.python import get_current_context
```

Comment:
I don't think databricks operator should import function from python operator

Reply:
Does it make sense to remove this utility to the `taskinstance` module (where `_CURRENT_CONTEXT` is maintained) or a separate utility module?